app.yml

```diff
@@ -22,7 +22,7 @@ default_events:
   # - deployment_status
   # - fork
   # - gollum
-  # - issue_comment
+  - issue_comment
```

Comment:
Why was this modified?

Reply:
could u tell how the bot is still responding to commands like `/help`

Reply:
> could u tell how the bot is still responding to commands like `/help`

Just saying it doesn't work for me currently without that, you can setup a fresh installation and see

I think GitHub separated the issue comments from the issue events like tags so i had to enable that too

Reply:
yes it's used by default when initially configuring the app. the reason why ubiquibot is still responding is that we've updated app settings manually after the first configuration. agreed with @devpanther 's change.